packages/frontend/src/components/MetricQueryData/MetricQueryDataProvider.tsx

```diff
@@ -44,10 +39,10 @@ type MetricQueryDataContext = {
     metricQuery?: MetricQuery;
     underlyingDataConfig: MetricQueryDataConfig | undefined;
     isUnderlyingDataModalOpen: boolean;
-    openUnderlyingDataModel: (
-        value: ResultRow[0]['value'],
-        meta: TableColumn['meta'],
-        row: ResultRow,
+    openUnderlyingDataModal: (
+        item: Field | TableCalculation | undefined,
+        value: UnderlyingValue,
+        row: UnderlyingValueMap,
```

Comment:
everything above is just type changes to make underlying data/drill into compatible with new pivot table